src/freenas/usr/local/sbin/disk_resize

```diff
@@ -45,9 +45,13 @@ else
 	echo "Resizing ${dev} to full capacity"
 fi
 
-# ATA and SCSI have completely different resize commands.
+# ATA, SCSI and NVMe have completely different resize commands.
 # SAT we handle as ATA via pass-through, but with some complications.
-devtype=`camcontrol devtype ${dev} 2>/dev/null`
+if nvmecontrol nsid ${dev} >/dev/null 2>/dev/null; then
+	devtype="nvme";
```

Comment:
Minor nit but seems to be a trailing semicolon here when not needed.

Reply:
Fixed.